gno.land/pkg/gnoweb/frontend/css/tx.config.js

```diff
@@ -39,10 +41,22 @@ export default {
         600: "#54595D", // Content color
         800: "#131313", // Background dark color
         900: "#080809", // Title color
+        // dark mode grays
+        d50: "#121212",    // Softer than pure black for backgrounds
```

Comment:
Because we use the palette directly in Tailwind (without separate utility tokens), it’s best to keep things consistent. We can merge our light and dark colors into a single, graded scale (e.g., 50 to 900) so that the dark: variants simply invert the shades. If a palette gets too crowded or if different colors serve distinct purposes, we can split them by family (like grays and greens) to maintain clarity and ease of use.

Reply:
I can handle the palette updates myself if you grant me access to your repository. It might be quicker and easier than multiple back-and-forth iterations. 👍